src/cmd/info.rs

```diff
@@ -43,6 +43,13 @@ fn print_public_key(public_key: &PublicKey, format: OutputFormat) -> Result {
             table.add_row(row!["Address", public_key.to_string()]);
             table.add_row(row!["Network", public_key.key_tag().network]);
             table.add_row(row!["Type", public_key.key_tag().key_type]);
+
+            // if this is an ED25519 key type, provide the Solana address
+            if public_key.key_tag().key_type == helium_crypto::KeyType::Ed25519 {
```

Comment:
you could `if let Ok(solana_key) = solana_sdk::pubkey::Pubkey::try_from(public_key.clone()) {...` 

Otherwise the `?` at the end of the solana_key construct would cause it to error out